prosody/rootfs/etc/cont-init.d/10-config

```diff
@@ -29,6 +29,9 @@ if [[ "$(stat -c %U /prosody-plugins-custom)" != "prosody" ]]; then
     chown -R prosody /prosody-plugins-custom
 fi
 
+grep -q muc_allowners <<< ${XMPP_MUC_MODULES} \
+&& patch -d /usr/lib/prosody/modules/muc -p0 < /prosody-plugins/muc_owner_allow_kick.patch
```

Comment:
why don't we unconditionally patch it? I know it only has an effect if that plugin is enabled, but it would simplify things, and we can do it at container build time.

Reply:
I agree with you. Moved to Docker file for unconditional patching in 65e3c89